deployments/notifyRewardAmount/xDAI/fox_regen_distribute.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-use-before-define */
+const hre = require("hardhat");
+const { ethers } = hre;
+
+const pools = [
+    {
+        address: "0x502EC7a040F486EE6Cb7d634D94764874B29dE68",
+        amount: "327387.1731240768",
+    }, // UNI (HNY/GIV)
+];
+
+// Two decimals of precision -> 760 = 7.60
+const distro = [
+    769, 769, 769, 769, 769, 769, 769, 769, 769, 769, 769, 769, 769,
```

Comment:
Where do these numbers come from? 769 * 13 == 9997, so if they're supposed to sum to 10000 I'd suggest bumping the first or last 3 by one each.